packs/rollable-tables/1st-level-consumables.json

```diff
@@ -1,8 +1,8 @@
 {
-    "_id": "tlX5PLwar8b1tmiQ",
-    "description": "Table of 1st-Level Consumables",
+    "_id": "ymDhLpTQPaZLqc7O",
```

Comment:
We might want to preserve the ids?

Reply:
Sorry, I created new tables without thinking about the fact that the old IDs should be reserved. I apologize, I should have thought about that more than I did.

Reply:
Should be good to go.